config_files/public/docker_test_config.ini

```diff
@@ -69,4 +69,7 @@ host_v2 = https://api.testwyre.com/v2/
 host    = https://api.testwyre.com/v3/
 
 [POLIPAYMENTS]
-host = https://poliapi.apac.paywithpoli.com/api
\ No newline at end of file
+host = https://poliapi.apac.paywithpoli.com/api
+
+[INTERCOM]
+app_id = kowgw7cm
```

Comment:
is this ok to have public @tristanhcole?

Reply:
Yes

It's like a heap or GA analytics code https://community.intercom.com/t/intercom-app-id-security/966

Reply:
> LGTM! Assume that app id is ok to be public?

https://github.com/teamsempo/SempoBlockchain/pull/413#discussion_r466784060

Reply:
Greeeat